app/assets/javascripts/tylium/modules/issues.js.coffee

```diff
@@ -3,3 +3,4 @@
 //= require tylium/modules/issues/merge
 //= require tylium/modules/issues/table
 //= require tylium/modules/issues/tag-input
+//= require tylium/modules/issues/local_tag_auto_save
```

Comment:
Why are you splitting this functionality in a separate class? Now we have to files concerned with `localStorage`, `auto_save/local` and this.

Reply:
This is _specifically_ to restore the selected tag dropdown in `isses/_form`, which is why i separated it out because it doesn't make sense to put it in `auto_save/local`.

Reply:
I know what the code does, I'm against peppering the `localStorage` code, for the same functionality (the Issues#form) to be spread across multiple files.

Reply:
Okay, sure, I've moved it to `auto_save/local` already